### PR TITLE
[WIP] Added support for 5.7 build

### PR DESCRIPTION
**Pull request by sshveta**
Made some changes to test 5.7 builds.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 4c0868fc51703aecda98070fda3fc7bdf1fe825c:

`widgetastic_manageiq/__init__.py`:
- :bangbang: Line 1281:101: **E501** *line too long (108 > 100 characters)*
- :bangbang: Line 1285:101: **E501** *line too long (108 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by sshveta** [comment]
One failure in 5.8 for sporadic catalog create validation message mismatch.

File: cfme/services/dashboard/ssui.py [hunk 1]
```diff
@@ -25,7 +27,7 @@ def in_dashboard(self):
 
     @property
     def is_displayed(self):
-        return self.in_dashboard
+        return self.in_dashboard()
```
Comment (quarckster) [inline cfme/services/dashboard/ssui.py]: Why did it become a method?

File: widgetastic_manageiq/__init__.py [hunk 2]
```diff
@@ -1168,8 +1168,14 @@ class SSUIlist(Widget, ClickableMixin):
...
-        list_item = Text(ParametrizedLocator('.//div[@id={list_name|quote}]/../div'
-            '//a[normalize-space(.)={item_name|quote}]'))
+        list_item = VersionPick({
+            Version.lowest():
+                Text(ParametrizedLocator('.//div[@id={list_name|quote}]/../div'
+                                         '//span[normalize-space(.)={item_name|quote}]')),
+            '5.8':
+                Text(ParametrizedLocator('.//div[@id={list_name|quote}]/../div'
+                                         '//a[normalize-space(.)={item_name|quote}]'))
+        })
```
Comment (quarckster) [inline widgetastic_manageiq/__init__.py]: **REQUIRED**
`list_item = Text(ParametrizedLocator('.//div[@id={list_name|quote}]/../div//*[normalize-space(.)={item_name|quote}]'))`

**Comment by sshveta** [comment]
Closed this by mistake .
New PR https://github.com/ManageIQ/integration_tests/pull/5741

**Pull request closed without merging**